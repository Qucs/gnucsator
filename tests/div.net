
Vac:V1 _net0 gnd U="1 V" f="" Phase="0" Theta="0"
R:R1 _net1 _net0 R="50" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
R:R2 _net1 gnd R="50" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
.TR:TR1 Type="lin" Start="0" Stop="1" Points="10" IntegrationMethod="Trapezoidal" Order="2" InitialStep="1 ns" MinStep="1u" MaxIter="150" reltol="0.001" abstol="1 pA" vntol="1 uV" Temp="26.85" LTEreltol="1e-3" LTEabstol="1e-6" LTEfactor="1" Solver="CroutLU" relaxTSR="no" initialDC="yes" MaxStep="0"
