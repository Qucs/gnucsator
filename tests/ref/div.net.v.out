
Vac #(.Phase(0),.Theta(0),.U(1)) V1 (.1(_net0),.2(0));
R #(.R(50),.Tc1(0.0),.Tc2(0.0),.Temp(26.85),.Tnom(26.85)) R1 (.p(_net1),.n(_net0));
R #(.R(50),.Tc1(0.0),.Tc2(0.0),.Temp(26.85),.Tnom(26.85)) R2 (.p(_net1),.n(0));
